Feature/ipv6 logging

geraldebberink opened the pull request:
I have added IPv6 support and the usage of the logging framework for the logging.

Tested on python 3.11.

thijsbrilleman committed:
Initial commit

geraldebberink committed:
hanged from cody.py to transip-dns.py so it also works in the PyCharm software

geraldebberink committed:
Changed to work for both IPv4 and IPv6.
And added the logging framework

thijsbrilleman reviewed (changes requested):
Code looks great!

Two small issues in .gitignore:

1. As you ignore dns.log, maybe set this filename to the default in the .py file?
2. specific.py is not in the repo; I guess it should be removed from the .gitignore.

Otherwise fine, tnx,!

GeraldEbberinkSaxion committed:
Removed specfic.py from .gitignore and made dns.log the default as per discussion.
https://github.com/thijsbrilleman/dynamic-ip-transip-automatic-update/pull/1#pullrequestreview-1433563638

GeraldEbberinkSaxion committed:
Added a changelog

thijsbrilleman commented:
Thanks!
